packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts

```diff
@@ -166,13 +163,24 @@ export interface StakerInfo {
   withdrawableAmount: bigint;
 }
 
-export interface IStatusEventFilter {
+export interface IStatusEventFilter extends IPagination {
   chainId: ChainId;
   statuses?: EscrowStatus[];
   from?: Date;
   to?: Date;
   launcher?: string;
+}
+
+export interface IWorker {
+  id: string;
+  address: string;
+  totalAmountReceived: number;
+  payoutCount: number;
+}
+
+export interface IWorkersFilter {
```

Comment:
Why don't you extend from IPagination here?
We might want to add orderDirection also.
The same in Python